frontend/app/model/FeatureChoice.scala

```diff
@@ -30,6 +32,8 @@ case object Books extends FeatureChoice {
   override val label = "4 free books"
 }
 case object FreeEventTickets extends FeatureChoice {
+  val allowance = zuoraFreeEventTicketsAllowance
   override val zuoraCode = "Events"
-  override val label = "6 free events"
+  override val label = s"$allowance free events"
+  val uom = "Events"
```

Comment:
This is [fully documented](https://knowledgecenter.zuora.com/BC_Developers/SOAP_API/E1_SOAP_API_Object_Reference/Usage) Zuora jargon, it stands for unit of measure